bootstrap/src/main/java/org/restcomm/media/bootstrap/ioc/provider/mgcp/MgcpChannelInitializerProvider.java

```diff
@@ -43,18 +45,31 @@
     private final MgcpMessageDecoder decoder;
     private final MgcpMessageEncoder encoder;
     private final MgcpChannelInboundHandler inboundHandler;
+    private final MgcpControllerConfiguration controller;
```

Comment:
Either store configuration POJO or mgcpBufferSize. There's no point keeping references to both.